examples/event_handling/pick_event_demo2.py

```diff
@@ -34,11 +34,11 @@
 def onpick(event):
 
     if event.artist != line:
-        return True
```

Comment:
This changes the return type and bool value of the return.   Do we not actually use this return?

Reply:
No event handlers use the return value; `CallbackRegistry` does not do anything with it.